pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -29,6 +31,7 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/util"
 	"istio.io/istio/pkg/log"
+	envoy_api_v2_core5 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
```

Comment:
can you call this as envoycore or something?

Reply:
yes